core/tests/test_utils.py

```diff
@@ -1266,7 +1266,27 @@ def get_gae_id_from_email(self, email):
         Returns:
             str. GAE ID of the user possessing the given email.
         """
-        return current_user_services.get_gae_id_from_email(email)
+
+        class _MockModelWithUser(datastore_services.Model):
+            """A simple model with a user property."""
+
+            _use_memcache = False
+            _use_cache = False
+            user = datastore_services.UserProperty(required=True)
```

Comment:
This still makes use of datastore_services.UserProperty. Could you please look into whether it is possible to get rid of this dependency altogether? (i.e. investigate the small number of tests that use it and see whether any of those usages are actually necessary)?

Reply:
Looks like we were able to mock it after all! :)